cmd/oras/internal/errors/errors.go

```diff
@@ -18,9 +18,43 @@ package errors
 import (
 	"fmt"
 
+	"github.com/spf13/cobra"
 	"oras.land/oras-go/v2/registry"
 )
 
+// Error is the error type for CLI error messaging.
+type Error struct {
```

Comment:
nit: Should we implement `Unwrap() error`?

Reply:
Currently there is no use case but worth doing since an error is already wrapped.